tests/e2e/Services/Storage/StorageConsoleClientTest.php

```diff
@@ -61,32 +73,33 @@ public function testGetStorageBucketUsage()
 
         // TODO: Uncomment once we implement check for missing bucketId in the usage endpoint.
 
-        // $response = $this->client->call(Client::METHOD_GET, '/storage/randomBucketId/usage', array_merge([
-        //     'content-type' => 'application/json',
-        //     'x-appwrite-project' => $this->getProject()['$id']
-        // ], $this->getHeaders()), [
-        //     'range' => '24h'
-        // ]);
+        $response = $this->client->call(Client::METHOD_GET, '/storage/randomBucketId/usage', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id']
+        ], $this->getHeaders()), [
+            'range' => '24h'
+        ]);
 
-        // $this->assertEquals($response['headers']['status-code'], 404);
+        $this->assertEquals($response['headers']['status-code'], 404);
 
         /**
          * Test for SUCCESS
          */
-        $response = $this->client->call(Client::METHOD_GET, '/storage/default/usage', array_merge([
+        $response = $this->client->call(Client::METHOD_GET, '/storage/' . $bucketId .  '/usage', array_merge([
             'content-type' => 'application/json',
             'x-appwrite-project' => $this->getProject()['$id']
         ], $this->getHeaders()), [
             'range' => '24h'
         ]);
 
         $this->assertEquals($response['headers']['status-code'], 200);
-        $this->assertEquals(count($response['body']), 6);
+        $this->assertEquals(count($response['body']), 7);
```

Comment:
Expected first, IIRC. We need to start fixing it at some point, you can be the first hero 😇

Reply:
Still not sure I wanna keep it in this PR, this PR is already huge?

Reply:
Agree, let's refactor in separate PR and let's make sure everyone in the team is aware, so all active PRs can be reviewed with this in mind.